tests/test_project.py

```diff
@@ -563,5 +572,36 @@ def helper_test_list_strain_maps(project):
             pytest.fail(str(e.str_itr()))
 
 
+def helper_test_add_project(project):
+    """Test add_project API"""
+
+    try:
+        project.add_project("", "", "")
+        pytest.fail("No exception raised when using an invalid parameter")
+    except SherlockAddProjectError as e:
+        assert str(e) == "Add project error: Project name cannot be blank"
+
+    if project._is_connection_up():
+        try:
+            project.add_project("Tutorial Project", "", "")
+            pytest.fail("No exception raised when creating a duplicate test")
+        except Exception as e:
+            assert type(e) == SherlockAddProjectError
+
+        try:
+            return_code = project.add_project(PROJECT_ADD_NAME, "", "")
+            assert return_code == 0
+            # Fix issue where api does not finish before returning
+            time.sleep(5)
```

Comment:
Change to delay for the minimum amount of time needed, in seconds. The other sleep() calls are one second which I think would be enough.